robotiq_hande_driver/src/communication.cpp

```diff
@@ -0,0 +1,81 @@
+#include "communication.hpp"
+
+#include <cstdio>
+#include <stdint.h>
+#include <string.h>
+
+
+namespace hande_driver  {
+
+constexpr auto kDeviceName = "/tmp/ttyUR";
+constexpr auto kBaudrate = 115200;
+constexpr auto kParity = 'N';
+constexpr auto kDataBits = 8;
+constexpr auto kStopBit = 1;
+constexpr auto kDebugModbus = true;
+constexpr uint8_t kSlaveID = 0x09;
+
+constexpr uint16_t kGripperOutputFirstReg = 0x07D0;
+constexpr uint16_t kGripperInputFirstReg = 0x03E8;
+
+Communication::Communication()
+// :    input_registers_{0, 0, 0}
+// ,    output_registers_{0, 0, 0}
+:    input_bytes_{0, 0, 0, 0, 0, 0}
+,    output_bytes_{0, 0, 0, 0, 0, 0}
```

Comment:
Since C++11 there are options for the guaranteed zeros initialization:
```suggestion
:    input_bytes_{}, output_bytes_{}
```
see https://en.cppreference.com/w/cpp/language/zero_initialization